Mandater/Utilities/CSVUtilities.cs

```diff
@@ -140,106 +154,108 @@ public static VDModel[] CsvToVdArray(string filePath)
-         /// <returns>An array of VDModel objects that can be used for simple in-memory queries or populating a database.</returns>
-        public static void CsvToCountyData(string filePath, Country country, ElectionContext context)
-        {
-            List<CountyData> objects = new List<CountyData>();
-            StreamReader file = new StreamReader(filePath);
-            string actualHeaderString = file.ReadLine(); // Skip
-            string currentLine;
-            while ((currentLine = file.ReadLine()) != null)
-            {
-                string[] objectFields = currentLine.Split(";");
-                if (objectFields.Length != 5)
-                {
-                    throw new CsvFileFormatException($"Found a line with length {objectFields.Length} instead of the required 4.", filePath, currentLine);
-                }
-                if (!int.TryParse(objectFields[0], out int year))
-                {
-                    throw new CsvFileFormatException("The field Year is not a valid integer.", filePath, currentLine);
-                }
-                if (!double.TryParse(objectFields[2], out double areal))
-                {
-                    throw new CsvFileFormatException("The field Areal is not a valid double.", filePath, currentLine);
-                }
-                if (!int.TryParse(objectFields[3], out int population))
-                {
-                    throw new CsvFileFormatException("The field Population is not a valid integer.", filePath, currentLine);
-                }
-                if (!int.TryParse(objectFields[4], out int seats))
-                {
-                    throw new CsvFileFormatException("The field Seats is not a valid integer.", filePath, currentLine);
-                }
+        //            Election currentObject = new Election
+        //            {
+        //                Year = year,
+        //                Algorithm = algorithm,
+        //                FirstDivisor = firstDivisor,
+        //                Threshold = threshold,
+        //                Seats = seats,
+        //                LevelingSeats = levelingSeats,
+        //                CountryId = country.CountryId,
+        //                Counties = new List<County>()
+        //            };
+        //            objects.Add(currentObject);
+        //        }
+        //        return objects.ToArray<Election>();
+        //    }
 
-                Election election = context.Elections.Where(e => e.Year == year).Single();
-                County county = election.Counties.Single(c => c.Name == objectFields[1]);
-                if (county == null)
-                {
-                    throw new CsvFileFormatException($"The field County does not match any known counties. {country.CountryId} - {objectFields[1]}", filePath, currentLine);
-                }
-                county.Seats = seats;
-            }
-        }
+        //    /// <summary>
```

Comment:
I recommend that in the future we use block comments for this kind of "commenting out", so that the representation of lines added is more accurate.